README.md

```diff
@@ -8,32 +8,33 @@ It consists of multiple packages and libraries centered around GraphQL AST.
 
 The core of this project is the GraphQL parser which parses GraphQL strings into an AST. The parsed AST can then be transformed, visited and printed back to GraphQL string.
 
-Another core part of the project is the types definitions used to execute GraphQL operations. They include AST-based operation, request with per operation context, and response. These types are used by the Links (see below).
-
 | Pub | Package | Library | Description |
 |-----|---------|---------|-------------|
 | [![version][package:gql:version]][package:gql] | [`package:gql`][package:gql:source] | `ast.dart` | GraphQL AST implementation with Visitor pattern and AST transformer |
 | [![version][package:gql:version]][package:gql] | [`package:gql`][package:gql:source] | `language.dart` | GraphQL source parser and printer. Recognizes both operations and SDL |
 | [![version][package:gql:version]][package:gql] | [`package:gql`][package:gql:source] | `document.dart` | Document and schema validation |
-| [![version][package:gql:version]][package:gql] | [`package:gql`][package:gql:source] | `execution.dart` | Implementation of types for GraphQL requests and responses |
 
 #### Code generation
 
-Current code generator is capable of building a Dart representation of GraphQL documents. It can be used to pre-build AST of `*.graphql` documents.
+GraphQL language enables code generation to speed up development and move document processing to build-time.
+  
+We provide code builders to generate Dart source code and file builders which easily plug into standard Dart code generation pipeline.
 
 Ideas for future work:
 - Dart types from a GraphQL schema
 - Dart types per GraphQL operation and it's variables (see [Artemis project][artemis])
```

Comment:
Should this be removed from "Ideas for future work"?

Reply:
🙇